### PR TITLE
Fix user.update() method

### PR DESCRIPTION
**Pull request by xezpeleta**
Hello,

I realized that user.update() method it's broken. That's why I can't make it work ( #61 ). This PR fixes the following issues:
- Fix patch_request header (requires Content-Type: application/json) #59
- Fix "Request body should be a valid JSON" #54 

Thanks

### CODE REVIEW

**Comment by prschmid** [comment]
@xezpeleta Thank you for putting together this PR!

Can you please make sure the tests pass?
https://travis-ci.org/github/actmd/zoomus/builds/668488649

**Comment by xezpeleta** [comment]
I updated the test but there is still some error (only with python 3.6 ?)

```
Oh no! 💥 💔 💥
1 file would be reformatted, 48 files would be left unchanged.
ERROR: InvocationError for command /home/travis/build/actmd/zoomus/.tox/black/bin/black --check . (exited with code 1)
___________________________________ summary ____________________________________
  py36: commands succeeded
ERROR:   black: commands failed
The command "tox" exited with 1.
Done. Your build exited with 1.
```

Sorry but I don't know how to fix it.

**Comment by tarkatronic** [comment]
Hey @xezpeleta! This just means that you need to run the `black` command against the code. 😄 If you look [here](https://github.com/actmd/zoomus/blob/master/CONTRIBUTING.md#code-formatting), there is a little bit of documentation on how to make that easier for you.

**Comment by tarkatronic** [comment]
Although I'm just now realizing that documentation needs a small update; you wouldn't use `pipenv` to install the `pre-commit` hooks. Instead, first make sure your virtualenv is activated and then run:

```sh
pre-commit install
```

**Comment by xezpeleta** [comment]
Thank you for teaching me fishing instead of giving me a fish :smile: @tarkatronic @prschmid

File: zoomus/components/user.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def create(self, **kwargs):
 
     def update(self, **kwargs):
         util.require_keys(kwargs, "id")
-        return self.patch_request("/users/{}".format(kwargs.get("id")), params=kwargs)
+        return self.patch_request("/users/{}".format(kwargs.get("id")), data=kwargs)
```
Comment (dltj) [inline zoomus/components/user.py:51]: Are you finding this is working for you?  I'm finding for the `/meetings/{}` endpoint I need:

```python
        return self.patch_request(
            "/meetings/{}".format(kwargs.get("id")), params=kwargs, data=kwargs["data"]
        )
```
...where the `data` parameter is explicitly set out of the `kwargs` dictionary.

Comment (mfutech) [reply zoomus/components/user.py:51]: yes the data=kwargs do not work for me. 
I used 
```
 def update(self, **kwargs):
        util.require_keys(kwargs, "id")
        id = kwargs.get("id")
        del kwargs["id"]
        return self.patch_request("/users/{}".format(id), **kwargs)
```

Comment (mfutech) [inline zoomus/components/user.py:51]: to keep the kwargs flexibility I used 
```suggestion
        id = kwargs.get("id")
        del kwargs["id"]
        return self.patch_request("/users/{}".format(id), **kwargs)
```
not happy with the del kwags["id"] but did not find better solution

Comment (xezpeleta) [reply zoomus/components/user.py:51]: Yes, it's working for me, but I don't know if it's the best solution. You can find a working example here: https://github.com/actmd/zoomus/issues/61#issuecomment-606466338

**Comment by prschmid** [comment]
Thank you for your patience! I have finally gotten around to making all of these changes and the updated code is now in develop as of https://github.com/actmd/zoomus/commit/fe272e32cda7b9ad40ca83814a49bac4ab185eff. I will be making a new release of the library shortly that contains all of these fixes. Thank you!

**Pull request closed without merging**